lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -17,6 +17,22 @@
 /** @typedef {LH.Artifacts.Rect} Rect */
 /** @typedef {{width: number, height: number}} Size */
 
+/**
+ * Returns whether rect2 is contained entirely within rect1;
```

Comment:
> Returns whether rect2 is contained entirely within rect1

should it not have a screenshot even if it's partially in bounds? e.g. what if just a few pixels are over the edge?

Reply:
eh, I'm inclined to keep it like this as it's simpler and this seems like an edge case (hehe).

Reply:
fair enough :)